templates/local_settings.py.j2

```diff
@@ -54,3 +70,13 @@ CATALOGUE = {
 
 MEDIA_ROOT = "/var/www/{{ app_name }}/uploaded"
 STATIC_ROOT = "/var/www/{{ app_name }}/static"
+
+ALLOWED_HOSTS = ['localhost', '{{ server_name }}']
```

Comment:
Do you mean those deployments aren't using a local_settings.py file at all?  Could you clarify?  I did do a minor refractor in https://github.com/GeoNode/geonode/commit/5018bd7183865d034822440db1cb6a844f31a8d8 to help decrease the need for local_settings.py